pshtt/pshtt.py

```diff
@@ -287,6 +287,8 @@ def basic_check(endpoint):
 
             endpoint.redirect_immediately_to = immediate
             endpoint.redirect_immediately_to_www = re.match(r'^https?://www\.', immediate)
+            if endpoint.redirect_immediately_to_www is not None:
+                endpoint.redirect_immediately_to_www = True
```

Comment:
what about changing 

```python
endpoint.redirect_immediately_to_www = re.match(r'^https?://www\.', immediate)
```

to:

```
endpoint.redirect_immediately_to_www = (re.match(r'^https?://www\.', immediate) is not None)
```

That seems slightly cleaner to me and more expressive, but I think it works fine either way.